TWLight/resources/models.py

```diff
@@ -432,4 +432,52 @@ class Meta:
 
     def __unicode__(self):
         # As with Stream, do not return the partner name here.
-        return self.full_name
\ No newline at end of file
+        return self.full_name
+
+
+
+class Suggest(models.Model):
+
+    class Meta:
+        app_label = 'resources'
+        verbose_name = 'suggestion'
+        verbose_name_plural = 'suggestions'
+        ordering = ['suggested_company_name']
+        
+    suggested_company_name = models.CharField(max_length=40,
+        # Translators: In the administrator interface, this text is help text for a field where staff can add partner suggestions.
+        help_text=_("Potential partner's name (e.g. McFarland)."))
+
+    description = models.TextField(blank=True, max_length=1000,
+        # Translators: In the administrator interface, this text is help text for a field where staff can provide a description of a potential partner.
+        help_text=_("Optional description of this potential partner."))
+
+    company_url = models.URLField(blank=True, null=True,
+        # Translators: In the administrator interface, this text is help text for a field where staff can link to a potential partner's website.
+        help_text=_("Link to the potential partner's website."))
+    
+    author = models.ForeignKey(User, related_name='suggest_author', blank=True, null=True, on_delete=models.SET_NULL,
+        # Translators: In the administrator interface, this text is help text for a field where staff can link a user as the author to a suggestion.
+        help_text=_("User who authored this suggestion."))
+    
+    plus_ones = models.ManyToManyField(User, related_name='suggest_likes', blank=True,
```

Comment:
I don't like the wording naming/help text/relationship here. Does this represent a user or a vote? Why are we switching between terminology for user votes (likes vs +1s)